src/server/schema/stripe.schema.ts

```diff
@@ -68,3 +68,8 @@ export type PaymentMethodDeleteInput = z.infer<typeof paymentMethodDeleteInput>;
 export const paymentMethodDeleteInput = z.object({
   paymentMethodId: z.string(),
 });
+
+export type ProductMetadata = z.infer<typeof productMetadataSchema>;
+export const productMetadataSchema = z.object({
+  vaultSizeKb: z.coerce.number().positive().optional(),
+});
```

Comment:
Might be a good idea to use `.passthrough()` at the end here since the metadata coming from stripe is pretty dynamic
```suggestion
}).passthrough();
```